src/frontend/src/catalog/system_catalog/mod.rs

```diff
@@ -283,34 +283,62 @@ pub fn get_sys_views_in_schema(schema_name: &str) -> Option<Vec<Arc<ViewCatalog>
         .map(Clone::clone)
 }
 
+/// The global registry of all builtin catalogs.
+pub static SYS_CATALOGS: LazyLock<SystemCatalog> = LazyLock::new(|| unsafe {
```

Comment:
What about only use unsafe when accessing `SYS_CATALOGS_INIT`? `unsafe` block looks scary to me.